asreview/io/ris_reader.py

```diff
@@ -14,28 +14,21 @@
 
 import logging
 
-import pandas as pd
+import pandas
 import rispy
-from rispy import TAG_KEY_MAPPING
-from rispy.config import LIST_TYPE_TAGS
 
-from asreview.config import COLUMN_DEFINITIONS
 from asreview.io.utils import standardize_dataframe
 
 
-RIS_KEY_LABEL_INCLUDED = "LI"
-
-
-def _tag_key_mapping(reverse=False):
-    # Add label_included into the specification and create reverse mapping.
-    TAG_KEY_MAPPING[RIS_KEY_LABEL_INCLUDED] = "included"
-    KEY_TAG_MAPPING = {TAG_KEY_MAPPING[key]: key for key in TAG_KEY_MAPPING}
-    for label in COLUMN_DEFINITIONS["included"]:
-        KEY_TAG_MAPPING[label] = "LI"
-    if reverse:
-        return KEY_TAG_MAPPING
+# Converter function for manipulating the internal "included" column
```

Comment:
Please make this into a docstring.

Reply:
There we go: https://github.com/asreview/asreview/pull/682/commits/3a756d9823ba82f337fb70f77a67d620327d9b41